packages/react-divider/src/components/Divider/useDividerStyles.ts

```diff
@@ -1,428 +1,240 @@
-    [
-      s => {
-        return !s.children;
-      },
-      {
-        '--divider-borderMargin': 0,
-      },
-    ],
-    /* Border color override */
-    [
-      s => s.color !== undefined,
-      { ':before': { '--divider-color': state.color }, ':after': { '--divider-color': state.color } },
-    ],
-    /* Appearance */
-    [
-      s => s.appearance === 'subtle',
-      tokens => ({
-        ':before': { '--divider-color': tokens.alias.color.neutral.neutralStroke3 },
-        ':after': { '--divider-color': tokens.alias.color.neutral.neutralStroke3 },
-      }),
-    ],
-    [
-      s => s.appearance === 'brand',
-      tokens => ({
-        '--divider-fontColor': tokens.alias.color.brand.brandBackgroundStatic,
-        ':before': { '--divider-color': tokens.alias.color.brand.brandBackgroundStatic },
-        ':after': { '--divider-color': tokens.alias.color.brand.brandBackgroundStatic },
-      }),
-    ],
-    [
-      s => s.appearance === 'strong',
-      tokens => ({
-        ':before': { '--divider-color': tokens.alias.color.neutral.neutralStroke1 },
-        ':after': { '--divider-color': tokens.alias.color.neutral.neutralStroke1 },
-      }),
-    ],
-    /* Horizontal Divider overrides */
-    [
-      s => !s.vertical,
-      {
-        width: '100%',
-        ':before': {
-          borderTopColor: 'var(--divider-color)',
-          borderTopWidth: 'var(--divider-borderSize)',
-          borderTopStyle: 'var(--divider-borderStyle)',
-        },
-        ':after': {
-          borderTopColor: 'var(--divider-color)',
-          borderTopWidth: 'var(--divider-borderSize)',
-          borderTopStyle: 'var(--divider-borderStyle)',
-        },
-        '&>:only-child': {
-          display: 'flex',
-          textAlign: 'center',
-        },
-      },
-    ],
-    /* Vertical Divider overrides */
-    [
-      s => s.vertical,
-      tokens => ({
-        //alignSelf: 'stretch',
-        minHeight: '20px',
-        flexDirection: 'column',
-        ':before': {
-          borderRightColor: 'var(--divider-color)',
-          borderRightWidth: 'var(--divider-borderSize)',
-          borderRightStyle: 'var(--divider-borderStyle)',
-        },
-        ':after': {
-          borderRightColor: 'var(--divider-color)',
-          borderRightWidth: 'var(--divider-borderSize)',
-          borderRightStyle: 'var(--divider-borderStyle)',
-        },
-        '&>:only-child': {
-          display: 'flex',
-          textAlign: 'center',
-        },
-      }),
-    ],
-    [
-      s => s.vertical && s.children !== undefined,
-      {
-        minHeight: '84px',
-      },
-    ],
+    ':after': {
+      display: 'flex',
+      flexGrow: 1,
+      boxSizing: 'border-box',
+    },
+  }),
+  childless: {
+    '--divider-borderMargin': 0,
+  },
+  subtle: tokens => ({
+    '--divider-color': tokens.alias.color.neutral.neutralStroke3,
+  }),
+  brand: tokens => ({
+    '--divider-fontColor': tokens.alias.color.brand.brandBackgroundStatic,
```

Comment:
wouldn't it rather make sense to have these as consts ? since you will use them later in the `var()`  statements

Reply:
Sure... that would avoid repetition, but since it's not used anywhere but by this component, I don't see reason for that

Reply:
it would still make maintenance easier if it ever came to it, there would be no safety at all changing `--divider-fontColor` to `--divider-font-color`